app/src/main/java/org/fossasia/susi/ai/skills/skilllisting/SkillListingFragment.kt

```diff
@@ -45,6 +48,15 @@ class SkillListingFragment : Fragment(), ISkillListingView, SwipeRefreshLayout.O
         swipe_refresh_layout.setOnRefreshListener(this)
         setUPAdapter()
         skillListingPresenter.getGroups(swipe_refresh_layout.isRefreshing)
+        /*
+         Adding onSwipe Listener for swipe between @see(ChatActivity.kt) and SkillsListingFragment
+         */
```

Comment:
Use '//' as it is a single line comment. Again, not a big issue, but please follow the coding style used in the project.